src/Common/src/Interop/Windows/secur32/ApplicationProtocolContext.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Net
+{
+    internal enum ApplicationProtocolNegotiationStatus
+    {
+        None = 0,
+        Success,
+        SelectedClientOnly
+    }
+
+    internal enum ApplicationProtocolNegotiationExt
+    {
+        None = 0,
+        NPN,
+        ALPN
+    }
+
+    // SecPkgContext_ApplicationProtocol
+    [StructLayout(LayoutKind.Sequential,CharSet = CharSet.Ansi)]
+    internal class ApplicationProtocolContext
```

Comment:
Do you mean including the SecPkgContext_ Prefix or simply the name should be "ApplicationProtocol" ?